src/modules/feature-limit/models/limit.model.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Column,
+  Model,
+  Table,
+  PrimaryKey,
+  DataType,
+  AllowNull,
+  BelongsToMany,
+} from 'sequelize-typescript';
+import { LimitTypes, LimitLabels } from '../limits.enum';
+import { TierModel } from './tier.model';
+import { TierLimitsModel } from './tier-limits.model';
+import { LimitAttributes } from '../limits.attributes';
+
+@Table({
+  underscored: true,
+  timestamps: true,
+  tableName: 'limits',
+})
+export class Limitmodel extends Model implements LimitAttributes {
+  @PrimaryKey
+  @Column(DataType.UUIDV4)
+  id: string;
+
+  @AllowNull(false)
+  @Column(DataType.STRING)
+  label: LimitLabels;
+
+  @AllowNull(false)
```

Comment:
I think a `@NotNull` decorator exists for this

Reply:
Yes, but the interpreter complains and the app does not run. The message asks you to use @AllowNull(false) instead of it. Pretty strange imo.